### PR TITLE
[4.1]Mail templates missing tags

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #37469
.

### Summary of Changes
Update the sql for new installs and updates to add the missing tags to the params. See #37469 for more information.

### CODE REVIEW

**Review by richard67 — APPROVED** [review]

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on ecbccd3f986d111665bd2109e3de8370e1862e9e<br /><br />Tested only on mysql.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37504">issues.joomla.org/tracker/joomla-cms/37504</a>.</sub>

File: administrator/components/com_admin/sql/updates/mysql/4.1.3-2022-04-07.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+UPDATE `#__mail_templates`
+   SET `params` = '{"tags":["message","date","extension","username"]}'
+ WHERE `template_id`= 'com_actionlogs.notification' AND `params` = '{"tags":["message","date","extension"]}';
+
+UPDATE `#__mail_templates`
+   SET `params` = '{"tags":["sitename","name","email","subject","body","url","customfields","contactname"]}'
+ WHERE `template_id`= 'com_contact.mail.copy' AND `params` = '{"tags":["sitename","name","email","subject","body","url","customfields"]}';
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/mysql/4.1.3-2022-04-07.sql]:
```suggestion
 WHERE `template_id` = 'com_actionlogs.notification' AND `params` = '{"tags":["message","date","extension"]}';

UPDATE `#__mail_templates`
   SET `params` = '{"tags":["sitename","name","email","subject","body","url","customfields","contactname"]}'
 WHERE `template_id` = 'com_contact.mail.copy' AND `params` = '{"tags":["sitename","name","email","subject","body","url","customfields"]}';
```

File: administrator/components/com_admin/sql/updates/postgresql/4.1.3-2022-04-07.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+UPDATE "#__mail_templates"
+   SET "params" = '{"tags":["message","date","extension","username"]}'
+ WHERE "template_id"= 'com_actionlogs.notification' AND "params" = '{"tags":["message","date","extension"]}';
+
+UPDATE "#__mail_templates"
+   SET "params" = '{"tags":["sitename","name","email","subject","body","url","customfields","contactname"]}'
+ WHERE "template_id"= 'com_contact.mail.copy' AND "params" = '{"tags":["sitename","name","email","subject","body","url","customfields"]}';
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/postgresql/4.1.3-2022-04-07.sql]:
```suggestion
 WHERE "template_id" = 'com_actionlogs.notification' AND "params" = '{"tags":["message","date","extension"]}';

UPDATE "#__mail_templates"
   SET "params" = '{"tags":["sitename","name","email","subject","body","url","customfields","contactname"]}'
 WHERE "template_id" = 'com_contact.mail.copy' AND "params" = '{"tags":["sitename","name","email","subject","body","url","customfields"]}';
```

**Comment by richard67** [comment]
I've restored the previous human test result in the issue tracker since the changes after that was only code style and a clean branch update.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fdd3e153a5d8e1b1655374b9191d88452f0ceb0b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37504">issues.joomla.org/tracker/joomla-cms/37504</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37504">issues.joomla.org/tracker/joomla-cms/37504</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**